shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -294,23 +294,22 @@ - (instancetype)initWithBridge:(fml::WeakPtr<flutter::AccessibilityBridgeIos>)br
     _bridge = bridge;
     _uid = uid;
     _children = [[NSMutableArray alloc] init];
-    _childrenInHitTestOrder = [[NSMutableArray alloc] init];
+    _childrenInHitTestOrder = [[NSArray alloc] init];
   }
 
   return self;
 }
 
 - (void)dealloc {
   for (SemanticsObject* child in _children) {
-    [child privateSetParent:nil];
+    child.parent = nil;
   }
   [_children removeAllObjects];
-  [_childrenInHitTestOrder removeAllObjects];
   [_children release];
   [_childrenInHitTestOrder release];
 
   _parent = nil;
-  _container.get().semanticsObject = nil;
```

Comment:
Just to make sure I'm reading the other code correctly: this is safe to remove because `semantics` object was changed to `weak` at some point, right, and thus we don't need to explicitly clear it any more?

Reply:
That's how I read it too, I don't think weak property would need to be set to nil in MRC?

I see this comment where it was introduced:
https://github.com/flutter/engine/pull/7244/files#r259211775

But I don't see why it would be needed...